### DISCUSSION TITLE
Is there a way to add a custom command for/through dance?

### DISCUSSION THREAD

**Discussion by leira** [71/dance#221] [General]
I'm trying to add a keybinding to toggle both the Activity Bar and the Side Bar. vscode only supports one command on one keybinding. There are extensions like multi-command can bind multiple commands on one keybinding. But I have a strong feeling dance might be able to do it, by define a custom command and bind it, I just don't know how. Is it really possible to add custom command with dance, and bind it?

**Comment by tshort** [comment]
See `dance.run`. It's great. Examples here: https://github.com/71/dance/wiki/Mirror-menu.

**Comment by leira** [comment]
Thanks, @tshort! It works great with the keybinding below:

```json
{
  "key": "ctrl+shift+b",
  "command": "dance.run",
  "args": {
    "commands": [
      "workbench.action.toggleActivityBarVisibility",
      "workbench.action.toggleSidebarVisibility"
    ]
  }
}
```
